be/app/src/main/java/com/deckbop/app/dao/DeckDAO.java

```diff
@@ -0,0 +1,55 @@
+package com.deckbop.app.dao;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.stereotype.Component;
+import org.springframework.dao.DataAccessException;
+import org.springframework.jdbc.support.rowset.SqlRowSet;
+import java.util.List;
+import org.springframework.security.core.context.SecurityContextHolder;
+import java.sql.Types;
+
+import com.deckbop.app.model.Deck;
+import com.deckbop.app.model.Card;
+import com.deckbop.app.controller.dto.DeckDTO;
+
+@Component
+public class DeckDAO {
+    
+    @Autowired
+    private JdbcTemplate jdbcTemplate;
+    
+    public void createDeck(DeckDTO deckDto){
+        Deck deck = new Deck();
+//        Object principal = SecurityContextHolder.getContext().getAuthentication().getPrincipal();
```

Comment:
we might want to discuss if we want to pull in the user id on the backend or as part of the request